spec/features/user/logged_in_users_see_classroom_tutorials_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+RSpec.describe "as a logged in user", :vcr do
+  before :each do
+    mod_3_tutorial_data = {
+      "title"=>"Back End Engineering - Module 3",
+      "description"=>"Video content for Mod 3.",
+      "thumbnail"=>"https://i.ytimg.com/vi/R5FPYQgB6Zc/hqdefault.jpg",
+      "playlist_id"=>"PL1Y67f0xPzdOq2FcpWnawJeyJ3ELUdBkJ",
+      "classroom"=>true,
+      "tag_list"=>["Internet", "BDD", "Ruby"],
+    }
+    m3_tutorial = Tutorial.create! mod_3_tutorial_data
+  end
+  it "can't see classroom tutorials if not logged in" do
```

Comment:
User is a regular visitor to the site and is not logged in and should not be able to see the classroom tutorial.